.github/workflows/pylake_test.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, "3.10"]
```

For conceptual symmetry :)
```suggestion
        python-version: ["3.7", "3.10"]
```